gateway/mw_go_plugin.go

```diff
@@ -139,6 +139,13 @@ func (m *GoPluginMiddleware) loadPlugin() bool {
 		m.Path = m.goPluginFromTykVersion(VERSION)
 	}
 
+	defer func() {
+		if e := recover(); e != nil {
+			err = fmt.Errorf("%v", e)
```

Comment:
just a little bit of comment here - Do we want to print the whole stack trace? Wouldn't be better to just print the plugin name/path that's failing?